holoviews/plotting/mpl/renderer.py

```diff
@@ -177,11 +176,11 @@ def diff(self, plot):
                 figure_format = self.params('fig').objects[0]
             else:
                 figure_format = self.fig
-            data = self.html(plot, figure_format, comm=False)
+            data = self.html(plot, figure_format)
         return data
 
 
-    def _figure_data(self, plot, fmt='png', bbox_inches='tight', **kwargs):
+    def _figure_data(self, plot, fmt='png', bbox_inches='tight', as_script=False, **kwargs):
```

Comment:
Need to document ``as_script`` appropriately. I assume it is about whether the return should be split into a separate JS script? How would you describe it?

Reply:
Correct. I'm not hugely happy shoving this onto ``_figure_data`` but it seemed like the only reasonable approach without duplicating a lot of code.